tensorflow/python/ops/image_ops.py

```diff
@@ -1004,6 +1007,44 @@ def adjust_contrast(images, contrast_factor):
     return convert_image_dtype(adjusted, orig_dtype, saturate=True)
 
 
+def adjust_gamma(image, gamma=1, gain=1):
+  """Performs Gamma Correction on the input image.
+    Also known as Power Law Transform. This function transforms the 
+    input image pixelwise according to the equation Out = In**gamma 
+    after scaling each pixel to the range 0 to 1.
+
+  Args:
+    image : A tensor.
+    gamma : A scalar. Non negative real number.
+    gain  : A scalar. The constant multiplier. 
+
+  Returns:
+    A tensor. Gamma corrected output image
+
+  Notes:
+    For gamma greater than 1, the histogram will shift towards left and
+    the output image will be darker than the input image.
+    For gamma less than 1, the histogram will shift towards right and
+    the output image will be brighter than the input image.
+
+  References:
+    [1] http://en.wikipedia.org/wiki/Gamma_correction
+  """
+
+  with ops.op_scope([image, gamma, gain], None, 'adjust_gamma') as name:
+    img = ops.convert_to_tensor(image, name='img', dtype=dtypes.float32)
+    image = ops.convert_to_tensor(image, name='image')
```

Comment:
Because `div` doesn't support `tf.int / tf.float`, I have to convert the `image.dtype` to `tf.float32`.